core/include/arch/thread_arch.h

```diff
@@ -56,6 +57,14 @@
 char *thread_arch_stack_init(thread_task_func_t task_func, void *arg, void *stack_start, int stack_size);
 
 /**
+ * @brief   interrupt usage
+ *
+ * While ISR is not a thread. its useful to be able to see
+ * stack usage for the interrupt context.
```

Comment:
not sure I understand this description... But if you change the brief description as proposed above, you can omit this.